### PR TITLE
Vet clinic: Database performance audit

### PR DESCRIPTION
**Pull request by luis-pomare**
In this milestone, we have:
- Added an `email` column to the `owners` table.
- Added tons of data to the table `visits` and the table owners for performance measurement.
- Added the necessary indexes and modify the tables to improve the performance of the following queries:
## First query
 `EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;`
### Before
image

### After indexing
image

### Before
image

### After indexing
image

### Before
image

### After indexing
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -166,3 +166,8 @@ JOIN vets ON vets.id = visits.vet_id
 WHERE vets.name = 'Maisy Smith'
 GROUP BY species.name
 ORDER BY COUNT(animals.id) DESC LIMIT 1;
+
+-- Queries for performance measurement
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
+EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';
```
Comment (Davidosky007) [inline queries.sql:173]: * Kindly improve the execution time for this query as the execution time on the PR screenshot can be lower

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @luis-pomare @dfsalam 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Well written PR 👍 
* Good readme file 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (luis-pomare) [reply queries.sql:173]: By adding a specific index for the target vet, we achieved a performance improvement of 50%

### Before
image

### After
image

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @luis-pomare and @dfsalam     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**